engine/src/main/resources/application.conf

```diff
@@ -36,6 +36,9 @@ system {
 
   // If 'true', a SIGINT will trigger Cromwell to attempt to abort all currently running jobs before exiting
   abort-jobs-on-terminate = false
+
+  // Max number of retries per job that the engine will attempt in case of a retryable failure received from the backend
+  maxRetries = 3
```

Comment:
That's a good point @mcovarr maybe the way to think about this is that as long as the backend is saying "this is retryable" the engine should retry it.  If the backend always returns retryable.. then we will retry forever... which would be a bad-backend.  And perhaps this "max" is really just a safety valve on that (although I would set it to like 10-20 or something).  The backend should know (a) the current attempt # and (b) the runtime attributes for a call.  So for the JES PBE it would decide that on failure # < runtime attribute for premptible it would return "retryable".  And then also when running if attempt # < premptible # (from runtime attributes) to launch a preemptible VM otherwise launch an on-demand one.

Maybe y'all already knew this and I'm just stating the obvious..